WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,21 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/regulations/wca-regulations.json")).freeze
+  rescue
+    REGULATIONS = [].freeze
```

Comment:
Yay! If it causes our builds to be significantly slower, though, maybe it's not worth it. Whatever you think makes the most sense.